docs/graphql/core/api-reference/schema-metadata-api/table-view.rst

```diff
@@ -304,6 +357,67 @@ Args syntax
      - :ref:`CustomColumnNames`
      - Customise the column fields
 
+.. _set_table_configuration:
+
+set_table_configuration
+-----------------------
+
+``set_table_configuration`` sets the configuration of a table, through which
+an identifier, custom root fields or custom column names of an already tracked
```

Comment:
Is it only one of those (identifier, custom root fields, custom column names) that can be set? Or actually all three?

Reply:
Oh, we can set all the three. I'll make the change.